uniprotkb-rest/src/main/java/org/uniprot/api/uniprotkb/controller/UniprotKBController.java

```diff
@@ -64,52 +73,124 @@ public UniprotKBController(
+                            @Content(
+                                    mediaType = APPLICATION_XML_VALUE,
+                                    array =
+                                            @ArraySchema(
+                                                    schema =
+                                                            @Schema(
+                                                                    implementation = Entry.class,
+                                                                    name = "entries"))),
+                            @Content(mediaType = TSV_MEDIA_TYPE_VALUE),
+                            @Content(mediaType = FF_MEDIA_TYPE_VALUE),
+                            @Content(mediaType = LIST_MEDIA_TYPE_VALUE),
+                            @Content(mediaType = XLS_MEDIA_TYPE_VALUE),
+                            @Content(mediaType = FASTA_MEDIA_TYPE_VALUE),
+                            @Content(mediaType = GFF_MEDIA_TYPE_VALUE)
+                        })
             })
     public ResponseEntity<MessageConverterContext<UniProtEntry>> searchCursor(
-            @Valid SearchRequestDTO searchRequest,
-            @RequestParam(value = "preview", required = false, defaultValue = "false")
+            @Valid @ModelAttribute SearchRequestDTO searchRequest,
+            @Parameter(hidden = true)
+                    @RequestParam(value = "preview", required = false, defaultValue = "false")
                     boolean preview,
-            @RequestHeader(value = "Accept", defaultValue = APPLICATION_JSON_VALUE)
-                    MediaType contentType,
             HttpServletRequest request,
             HttpServletResponse response) {
+
         setPreviewInfo(searchRequest, preview);
+        MediaType contentType = getAcceptHeader(this.request);
         QueryResult<UniProtEntry> result = entryService.search(searchRequest);
         return super.getSearchResponse(
                 result, searchRequest.getFields(), contentType, request, response);
     }
 
+    @Tag(name = "uniprotkb")
     @RequestMapping(
             value = "/accession/{accession}",
             method = RequestMethod.GET,
             produces = {
-                TSV_MEDIA_TYPE_VALUE, FF_MEDIA_TYPE_VALUE, LIST_MEDIA_TYPE_VALUE,
-                        APPLICATION_XML_VALUE,
-                APPLICATION_JSON_VALUE, XLS_MEDIA_TYPE_VALUE, FASTA_MEDIA_TYPE_VALUE,
-                        GFF_MEDIA_TYPE_VALUE
+                TSV_MEDIA_TYPE_VALUE,
+                FF_MEDIA_TYPE_VALUE,
+                LIST_MEDIA_TYPE_VALUE,
+                APPLICATION_XML_VALUE,
+                APPLICATION_JSON_VALUE,
+                XLS_MEDIA_TYPE_VALUE,
+                FASTA_MEDIA_TYPE_VALUE,
+                GFF_MEDIA_TYPE_VALUE
+            })
+    @Operation(
+            summary =
+                    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua",
```

Comment:
For the summaries / descriptions throughout, it might be a "nice to have" if it was possible to read the contents from a file. The benefit is that it clearly separates description from code -- so that potentially a non-developer could update it. (Also it makes the code smaller :) ).